xapian-core/api/registry.cc

```diff
@@ -105,7 +105,7 @@ register_object(map<string, T*> & registry, const T & obj)
 
     r.first->second = clone;
 }
- 
+
```

Comment:
It's generally unhelpful to include whitespace changes (even if they are to conform to what's expected) amongst functional changes as it makes those functional changes harder to review.  So please avoid doing this in future patches - send a separate patch/PR for the whitespace cleanup instead.